semantic/flatbuffers.go

```diff
@@ -570,6 +570,16 @@ func (e *FunctionExpression) FromBuf(fb *fbsemantic.FunctionExpression) error {
 		return err
 	}
 
+	fbVectorized := fb.Vectorized(nil)
+	if fbVectorized == nil {
+		return errors.New(codes.Internal, "missing function vectorized")
```

Comment:
I would expect this function to be optional. so it should not be an error if it's not there.